File: restaurant-server/src/main/java/com/cm/restaurant_server/business/domain/entity/MesaRestaurante.java
package com.cm.restaurant_server.business.domain.entity;

import com.cm.restaurant_server.business.domain.enumeration.EstadoMesa;
import jakarta.persistence.Entity;
import jakarta.persistence.EnumType;
import jakarta.persistence.Enumerated;
import lombok.*;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class MesaRestaurante extends Base {
    private int identificadorMesa;
    @Enumerated(EnumType.STRING)
    private EstadoMesa estadoMesa;
    private int capacidadPersonas;
    private String zonaFisica;
}

File: dashboard/src/types/mesaRestaurante.ts
export enum EstadoMesa {
  LIBRE = "LIBRE",
  RESERVADA = "RESERVADA",
  OCUPADA = "OCUPADA",
  FUERA_DE_SERVICIO = "FUERA_DE_SERVICIO",
}

export interface MesaRestaurante {
  id: string;
  identificadorMesa: number;
  estadoMesa: EstadoMesa;
  capacidadPersonas: number;
  zonaFisica: string;
}
